script/package-version/package-version.php

```diff
@@ -32,7 +32,8 @@
 
   header('Link: <' . KeymanHosts::Instance()->api_keyman_com . '/schemas/package-version.json#>; rel="describedby"');
 
-  $params = fix_array_params($_SERVER['QUERY_STRING']);
+  $env = getenv();
+  $params = fix_array_params($env['QUERY_STRING']);
```

Comment:
standard prop

Reply:
```suggestion
  $env = getenv();
  $params = fix_array_params($_SERVER['QUERY_STRING']);
```

Reply:
Reverted in https://github.com/keymanapp/api.keyman.com/pull/213/commits/0ee70c7dd56af88308d3d466b1524eb153e08903